tests/e2e/provisioning/test/e2e_provisioning_test.go

```diff
@@ -46,4 +50,39 @@ func Test_E2E_Provisioning(t *testing.T) {
 
 	err = ts.dashboardChecker.AssertRedirectedToUAA(dashboardURL)
 	assert.NoError(t, err)
+
+	if ts.IsTestAzureEventHubsEnabled {
+		checkAzureEventHubProperties(ts, t, operationID)
+	}
+}
+
+func checkAzureEventHubProperties(ts *Suite, t *testing.T, operationID string) {
+	filter := fmt.Sprintf("tagName eq 'InstanceID' and tagValue eq '%s'", ts.InstanceID)
+	groupListResultPage, err := (*ts.azureClient).ListResourceGroup(context.TODO(), filter, nil)
+
+	assert.NoError(t, err)
+	assert.Equal(t, http.StatusOK, groupListResultPage.Response().StatusCode, "HTTP GET fails for ListResourceGroup")
+	assert.Equal(t, 1, len(groupListResultPage.Values()), "groupListResultPage should return 1 ResourceGroup")
+
+	rg := groupListResultPage.Values()[0]
+	tagsRG := groupListResultPage.Values()[0].Tags
+
+	// Check tags for ResourceGroup
+	assert.Equal(t, operationID, *tagsRG[azure.TagOperationID], "Tag OperationID for ResourceGroup is incorrect")
+	assert.Equal(t, ts.brokerClient.InstanceID(), *tagsRG[azure.TagInstanceID], "Tag InstanceID for ResourceGroup is incorrect")
+	assert.Equal(t, ts.brokerClient.SubAccountID(), *tagsRG[azure.TagSubAccountID], "Tag SubAccountID for ResourceGroup is incorrect")
+
+	ehNamespaceListResultPage, err := (*ts.azureClient).ListEHNamespaceByResourceGroup(context.TODO(), *rg.Name)
```

Comment:
Probably `rg.Name` will never be nill if a successful response returned, but can we still check for nil here? :)

Reply:
checking for nil now